### DISCUSSION TITLE
Consider taking inspiration from https://github.com/radian-software/radian 's emacs?

### DISCUSSION THREAD

**Discussion by hab25** [SystemCrafters/crafted-emacs#267] [Ideas]
Hi @jeffbowman @daviwil, in case you're not aware, [radian](https://github.com/radian-software/radian) is MIT-licensed and contains an emacs distribution that seems to highly align with crafted-emacs as evidenced by these selections from radian's summary:
1) > [aggressively using best practices](https://github.com/radian-software/radian#:~:text=aggressively%20using%20best%20practices)
2) > [extensively documenting and commenting all code](https://github.com/radian-software/radian#:~:text=extensively%20documenting%20and%20commenting%20all%20code)
3) > [remaining as simple as possible](https://github.com/radian-software/radian#:~:text=remaining%20as%20simple%20as%20possible)
4)  > [supporting local configuration without the need to fork this repository](https://github.com/radian-software/radian#:~:text=supporting%20local%20configuration%20without%20the%20need%20to%20fork%20this%20repository)

So far I have yet to see any other emacs project (*maybe* excluding emacs itself) come even remotely close to the level of effort put into best practices. The most impressive thing, and that I've seen no other project do (including all of the ELPA)s, is that radian is
> [Tested on GitHub Actions with Docker configuration included for all supported Emacs versions](https://github.com/radian-software/radian#:~:text=Tested%20on%20GitHub%20Actions%20with%20Docker%20configuration%20included%20for%20all%20supported%20Emacs%20versions)

Further aiding the point that radian cares a lot about correctness and best practices, radian's maintainer made:
[el-patch](https://github.com/radian-software/el-patch)
[straight.el](https://github.com/radian-software/straight.el)

Both implement mechanisms necessary to make emacs configurations not rot (i.e., be future-proof) .

Radian also seems to put more effort than any other project I've seen in issue organization; see their [extentensively labeled issue tracker](https://github.com/radian-software/radian/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).

**Comment by jeffbowman** [comment]
Thanks for sharing your thoughts.  Crafted Emacs and Radian have several things in common, but Crafted Emacs will be changing shortly to align more closely with the original intention of providing a faster method to building your own configuration.  Others kits, like Prelude, Doom, Spacemacs, and probably Radian try to provide a working solution out of the box.  While, historically, Crafted Emacs has done approximately the same thing, the intention (and you can see this in the read me and the pinned issue talking about goals and direction, etc) has always been to be something a user can use to **build** their own configuration, ideally, eventually dropping Crafted Emacs as their configuration becomes more "their own". 

In 2023, my goal is to get much much closer to that intention, and I've posted a blog about it [here](https://write.as/7c54acetdzbkmtu9.md). 

Radian is certainly a good project and worth drawing some inspiration from, but our goals are currently shifting, so we'll see how much inspiration comes from not just this but other starter kits et.al. going forward.

**Reply by hab25** [reply]
Thanks for replying; the new direction looks good! Doom is also going for standalone modules, and I'm looking forward to being able to combine the best tools for the job.
